seata-spring-autoconfigure/seata-spring-autoconfigure-server/src/main/java/io/seata/spring/boot/autoconfigure/properties/server/ServerRecoveryProperties.java

```diff
@@ -27,15 +27,44 @@
 @ConfigurationProperties(prefix = SERVER_RECOVERY_PREFIX)
 public class ServerRecoveryProperties {
 
-    private Integer handleAllSessionPeriod = 1000;
+    private Integer committingRetryPeriod = 1000;
```

Comment:
why Integer not int ?

Reply:
I just copy old properties,I will change it to int